ArchiSteamFarm/IPC/Startup.cs

```diff
@@ -97,44 +98,47 @@ internal sealed class Startup {
 		// Add support for default root path redirection (GET / -> GET /index.html), must come before static files
 		app.UseDefaultFiles();
 
-		// Add support for static files (e.g. HTML, CSS and JS from IPC GUI)
-		app.UseStaticFiles(
-			new StaticFileOptions {
-				OnPrepareResponse = static context => {
-					if (context.File is { Exists: true, IsDirectory: false } && !string.IsNullOrEmpty(context.File.Name)) {
-						string extension = Path.GetExtension(context.File.Name);
-
-						CacheControlHeaderValue cacheControl = new();
-
-						switch (extension.ToUpperInvariant()) {
-							case ".CSS" or ".JS":
-								// Add support for SRI-protected static files
-								// SRI requires from us to notify the caller (especially proxy) to avoid modifying the data
-								cacheControl.NoTransform = true;
-
-								goto default;
-							default:
-								// Instruct the caller to always ask us first about every file it requests
-								// Contrary to the name, this doesn't prevent client from caching, but rather informs it that it must verify with us first that his cache is still up-to-date
-								// This is used to handle ASF and user updates to WWW root, we don't want from the client to ever use outdated scripts
-								cacheControl.NoCache = true;
-
-								// All static files are public by definition, we don't have any authorization here
-								cacheControl.Public = true;
-
-								break;
-						}
+#if !NETFRAMEWORK && !NETSTANDARD
```

Comment:
Is totally ignoring netf variant a good idea? I can understand argument that nobody bothers to create plugins for netf, Even I don't bother to support it in new plugins, but technically plugins for netf are still supported, so maybe we need some workaround for those, just for the sake of compatibility? @JustArchi what do you think about it?

Reply:
@fazelukario where did you get that info? I can see that the API supports the same syntax: <https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.staticfileoptions?view=aspnetcore-2.0>

Reply:
> @fazelukario where did you get that info? I can see that the API supports the same syntax: https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.staticfileoptions?view=aspnetcore-2.0

@JustArchi I'm not sure about this, but according to the docs, the "Serve files from different locations" section only appears in asp.net 6.
But we can try if you help me test it on netf.

Reply:
Just making it compile on netf will do, but since it's pain in the a** to work with, I can add compat (if possible) in follow-up after the PR, up to you.

Reply:
Only thing that prevents it from compiling is `Directory.GetParent`. It may be either added to `JustArchiNET.Madness` or replaced with alternatives (like I did for testing). Other than that it compiles and works fine.

Reply:
> Just making it compile on netf will do, but since it's pain in the a** to work with, I can add compat (if possible) in follow-up after the PR, up to you.

@JustArchi If by compat you mean compatibility, then yes it would be great.
I don't know much about netf as you use it, so I'd be grateful if you could take care of compatibility after my PR.

Reply:
ASF targets .NET Framework 4.8.1 for netf variant, and while Microsoft indeed lists this method as available for this framework in documentation, in reality I'm getting `error CS0117: 'Directory' does not contain a definition for 'GetParent'` error message. Maybe it's because `JustArchiNET.Madness` don't have it listed?